test/appium/tests/atomic/chats/test_activity_center.py

```diff
@@ -0,0 +1,251 @@
+
+        self.home_1.just_fyi("Verify there are still no chats after relogin")
+        self.home_1.relogin()
+        if self.home_1.element_by_text_part(self.username_2[:20]).is_element_displayed(2):
+            self.errors.append("1-1 chat appears on Chats view after relogin")
+        if self.home_1.element_by_text_part(self.group_chat_name_1).is_element_displayed(2):
+            self.errors.append("Group chat appears on Chats view after relogin")
+        self.home_1.notifications_button.click()
+        if self.home_1.element_by_text_part(self.username_2[:20]).is_element_displayed(2):
+            self.errors.append("1-1 chat request reappears back in Activity Center view after relogin")
+        if self.home_1.element_by_text_part(self.group_chat_name_1).is_element_displayed(2):
+            self.errors.append("Group chat request reappears back in Activity Center view after relogin")
+
+        self.errors.verify_no_errors()
+
+    @marks.testrail_id(702184)
+    def test_activity_center_accept_chats(self):
+
+        [home.home_button.double_click() for home in [self.home_1, self.home_2]]
+
+        self.device_2.just_fyi('Device2 creates 1-1 and Group chat again')
+        self.home_2.get_chat_from_home_view(self.username_1).click()
+        self.device_2_one_to_one_chat.send_message(self.message_from_sender)
+        self.device_2_one_to_one_chat.home_button.double_click()
+        self.home_2.create_group_chat([self.username_1], group_chat_name=self.group_chat_name_2)
+
+        self.device_1.just_fyi('Device1 accepts both chats (via Select All button) and verifies they disappeared '
+                               'from activity center view but present on Chats view')
+        self.home_1.notifications_button.click()
+        self.home_1.notifications_select_button.click()
+        self.home_1.notifications_select_all.click()
+        self.home_1.notifications_accept_and_add_button.click()
+
+        if self.home_1.element_by_text_part(self.username_2[:20]).is_element_displayed(2):
+            self.errors.append("1-1 chat request stays on Activity Center view after it was accepted")
+        if self.home_1.element_by_text_part(self.group_chat_name_2).is_element_displayed(2):
+            self.errors.append("Group chat request stays on Activity Center view after it was accepted")
+        self.home_1.home_button.double_click()
+
+        if not self.home_1.element_by_text_part(self.username_2[:20]).is_element_displayed(2):
+            self.errors.append("1-1 chat is not added on home after accepted from Activity Center")
+        if not self.home_1.element_by_text_part(self.group_chat_name_2).is_element_displayed(2):
+            self.errors.append("Group chat is not added on home after accepted from Activity Center")
+
+        self.errors.verify_no_errors()
+
+    @marks.testrail_id(702185)
+    def test_activity_center_notifications_on_mentions_in_groups_and_communities_empty_state(self):
```

Comment:
better to remove creating community from group of tests and add simple check to the community [695842] itself